view.go

```diff
@@ -181,7 +210,81 @@ func (v *View) Refresh(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	return seqStore.Set(ctx, seq)
+	if seq == lastSeq {
+		return nil
+	}
+	err = seqStore.Set(ctx, seq)
+	if err != nil {
+		return err
+	}
+
+	if v.reducerTable == "" {
+		return nil
+	}
+
+	_, err = v.sqldb.ExecContext(ctx, `DROP TABLE `+v.reducerTable)
+	if err != nil {
+		return err
+	}
+	err = v.createReduceTable(ctx)
+	if err != nil {
+		return err
+	}
```

Comment:
Probably just DELETE FROM table. (doesn't matter that much though, since you won't do this in the next PR).